wagtail/test/testapp/models.py

```diff
@@ -2128,3 +2129,12 @@ class GenericSnippetNoFieldIndexPage(GenericSnippetPage):
         ContentType, on_delete=models.SET_NULL, null=True, blank=True
     )
     snippet_content_type_nonindexed.wagtail_reference_index_ignore = True
+
+
+class CustomPermissionTester(PagePermissionTester):
+    pass
+
+
+class CustomPolicyPage(Page):
```

Comment:
Maybe name this `CustomPermissionPage` instead so that we aren't tying it to `Policy` or `Tester` while we do further work on permissions.

```suggestion
class CustomPermissionPage(Page):
```